upi/openstack/bootstrap.yaml

```diff
@@ -20,6 +20,17 @@
       allowed_address_pairs:
       - ip_address: "{{ os_apiVIP }}"
 
+  - name: 'Create the bootstrap dualstack server port'
+    os_port:
+      name: "{{ os_port_bootstrap }}"
+      network: "{{ os_dualstack_network }}"
```

Comment:
What is `os_dualstack_network`? Shouldn't it be just `os_network`?

Reply:
This is just an draft, it will be update following your implementation of the dual-stack network and api/ingress ports